module-2/backend/week_two.md

```diff
@@ -26,20 +30,59 @@ end
 
 Now how would you find the owner of the team with an id of 4?
 
+> `Team.first.owner`
```

Comment:
This would only work if we had deleted teams 1-3. Need to get the Team with an id of 4 first.

```ruby
Team.find(4).owner
```